crucible-llvm/src/Lang/Crucible/LLVM/MemModel/Strings.hs

```diff
@@ -2,12 +2,51 @@
 -- deprecated imports into this module, and remove the exports from MemModel.
 {-# OPTIONS_GHC -Wno-warnings-deprecations #-}
 
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE ImplicitParams #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+
 -- | Manipulating C-style null-terminated strings
 module Lang.Crucible.LLVM.MemModel.Strings
   ( Mem.loadString
   , Mem.loadMaybeString
   , Mem.strLen
+  , storeString
   ) where
 
+import qualified Data.Parameterized.NatRepr as DPN
+import qualified Data.Vector as Vec
+import qualified Lang.Crucible.Backend as LCB
+import qualified Lang.Crucible.LLVM.DataLayout as CLD
+import qualified Lang.Crucible.LLVM.MemModel as LCLM
 import qualified Lang.Crucible.LLVM.MemModel as Mem
+import qualified What4.Interface as WI
 
+-- | Store a string to memory, adding a null terminator at the end.
+storeString ::
+  forall sym bak w.
+  ( LCB.IsSymBackend sym bak
+  , WI.IsExpr (WI.SymExpr sym)
```

Comment:
`IsExpr (SymExpr sym)` is a superclass of `IsSymBackend sym bak`, so the former constraint is redundant.